src/containers/LoginGate/LoginGate.scss

```diff
@@ -0,0 +1,53 @@
+.LoginGate {
+  & .header {
+    flex-direction: row;
+    padding: 3px;
+    z-index: $z-index-header;
+    top: 0;
+    left: 0;
+    background: $color-purple;
+  }
+
+  & > .container {
+    @include border-radius($base-border-radius);
+    position: relative;
+    z-index: $z-index-modal + 2;
+    width: 60%;
+    max-width: 1024px - 2 * $base-padding;
+    margin: auto;
+    background: $color-white;
+    border: $base-border;
+  }
+
+  & .logo-id {
+    height: 100px;
+
+    & img {
+      height: 100%;
+      width: auto;
+    }
+  }
+
+  & form {
```

Comment:
You should only style what is actually inside this component (LoginGate), not the other components, which are included. _Separation of Concerns_